script.js

```diff
@@ -14,3 +205,67 @@ backMenu.forEach((element) => element.addEventListener('click', () => {
 }));
 
 /* ---------------------hamburger code above----------------------------- */
+
+/* ---------------------Windows popup code below----------------------------- */
+const body = document.querySelector('body');
+const view = document.querySelectorAll('.view-project');
+view.forEach((element) => element.addEventListener('click', () => {
+  const show = document.createElement('div');
+  const detailPop = document.createElement('div');
+  show.className = 'show';
+  detailPop.className = 'pop';
+  detailPop.innerHTML = `
+                      <div class="popup-header">
+                        <h2 class="desktop" id="desktop">Keeping track of hundreds
+                         of components website</h2>
+                         <h2 class="mob-title">Multi Post Stories</h2>
+                        <span id="close">&times;</span>
+                      </div>
+                        <div class="popup-button">
+                          <button id="lists">HTML</button>
```

Comment:
- [ ] You have done a great job💪 of implementing the details pop-up window🖥 in each project, as suggested by the previous reviewer🙄. However, the data displayed in the pop-up window is static. You should populate the data dynamically in the pop-up window from an array, similar to how you populated the project list🎫. This will ensure that the details in your pop-up windows are valid.